### ISSUE TITLE
Dot.Notated Library import is not loaded

### ISSUE THREAD

**Issue by myjniak** [jnhyperion/HyperRobotFrameworkPlugin#64]
This is currently not supported, although it's completely legal (and useful) in Robot Framework:
```
*** Settings ***
Library    MyLibrary.MyModule    AS    MyModule
```

The complete example you can install and easily reproduce can be found here:
https://github.com/myjniak/hyper-robot-issue

Images and gifs in README.md can also help explaining my issue :)

**Comment by jnhyperion** [comment]
Thank you for your detailed explanation.

There're 2 issues here:

1. Importing library with alias, which is not currently supported, similar with: https://github.com/jnhyperion/HyperRobotFrameworkPlugin/issues/14

2. Library cannot be recognized without library absolute path.

The 2 library import statements are all valid based on robot import rules, but plugin cannot parse the library path correctly. I can confirm this is the bug.
``` robot
Library    MyLibrary.MyModule                       # cannot be recognized
Library    MyLibrary.MyModule.MyModule              # cannot be recognized
```

In your case, the workaround is to use library absolution path without any alias.

``` robot
*** Settings ***
Library    MyLibrary.MyModule.robot_interface.MyModule

*** Test Cases ***
Test 1
    My Super Keyword

```

**Comment by myjniak** [comment]
That's an almost perfect workaround for me :)
The problem is that I have another library with keyword "My Super Keyword"
So I need to use namespaces like so:

```robot
*** Settings ***
Library    MyLibrary.MyModule.robot_interface.MyModule
Library    AnotherLibrary

*** Test Cases ***
Test 1
    MyModule.My Super Keyword
    AnotherLibrary.My Super Keyword
```

which is not recognized by the plugin.
See: 
https://github.com/myjniak/hyper-robot-issue/blob/master/test/keywords_with_namespaces.robot

**Comment by jnhyperion** [comment]
the namespace you mentioned should be the absolute path:

``` robot
*** Settings ***
Library    MyLibrary.MyModule.robot_interface.MyModule
Library    AnotherLibrary

*** Test Cases ***
Test 1
    MyModule.My Super Keyword # robot will not find this kw
    MyLibrary.MyModule.robot_interface.MyModule.My Super Keyword # this works
    AnotherLibrary.My Super Keyword
```

Plugin does not support access kw with library absolute path, will plan this feature in the next version :)

**Comment by jnhyperion** [comment]
fixed in `0.1.14`

**Comment by myjniak** [comment]
Thank you so much, that was a really fast fix 👍
